bin/tronctl

```diff
@@ -129,7 +135,33 @@ def event_discard(args):
         )
 
 
-def control_objects(args):
+def _get_triggers_for_action(server: str, action_identifier: str) -> Optional[Tuple[str, ...]]:
+    try:
+        namespace, job_name, run_number, action_name = action_identifier.split('.')
+    except ValueError:
+        print(f"Unable to fully decompose {action_identifier}: expected an identifier of the form (namespace).(job).(run).(action)")
+        return None
+
+    trigger_response = client.request(
+        uri=urljoin(
+            server,
+            f"/api/jobs/{namespace}.{job_name}/{run_number}/{action_name}",
+        ),
+    )
+    if trigger_response.error:
+        print(f"Unable to fetch downstream triggers for {action_identifier}: {trigger_response.error}")
+        return None
+
+    # triggers are returned by the API as comma-separated values with a space after every comma, which is
+    # not automation-friendly - thus the string-manipulation
+    triggers = trigger_response.content.get('trigger_downstreams', '').replace(" ", "").split(',')
```

Comment:
I think `.split(', ')` would do the same thing, so you can skip the replace.

Reply:
oh true, thanks!